data/society/openstreetmap-places/index.html

```diff
@@ -60,32 +60,32 @@ <h4 id="DataCreation">Data Creation Process</h4>
         via point-in-polygon and near analyses.  A numeric check is done on the 'name' field to remove features where the name is 
         less than 3 characters long or more than 50% numeric characters.  This eliminates several features derived from 
         the buildings layer where the 'name' is simply an apartment complex building number (ex: 3A) or house number 
-        (ex: 1612).  Finally, additional attributes (osm_addr, opening_hours, phone, website, cuisine, etc.) are pulled 
+        (ex: 1612).  Finally, additional attributes (osm_addr, open_hours, phone, website, cuisine, etc.) are pulled 
         from the <a href="https://wiki.openstreetmap.org/wiki/Overpass_API">Overpass API</a> and joined to the filtered data using 
         the 'osm_id' field as the join key.
     </p>
     <h4 id="FieldDescriptions">Field Descriptions</h4>
     <ul>
-        <li>addr_dist - the distance (m) to the nearest UGRC address point within 25 m
-        <li>osm_id - the feature ID in the OSM database
-        <li>category - the feature's data class based on the 4-digit code and tags in the OSM database
-        <li>name - the name of the feature in the OSM database
-        <li>county - the county the feature is located in (assigned from UGRC's county boundaries)
-        <li>city - the city the feature is located in (assigned from UGRC's municipal boundaries)
-        <li>zip - the zip code of the feature (assigned from UGRC's approximation of zip code boundaries)
-        <li>block_id - the census block the feature is located in (assigned from UGRC's census block boundaries)
-        <li>ugrc_addr - the nearest address (within 25 m) from the UGRC address point database
-        <li>disclaimer - a note from UGRC about the ugrc_addr field
-        <li>lon - the approximate longitude of the feature, calculated in WGS84 [EPSG:4326] (for the most precision, it is recommended that the feature's geometry is used instead of this field)
-        <li>lat - the approximate latitude of the feature, calculated in WGS84 [EPSG:4326] (for the most precision, it is recommended that the feature's geometry is used instead of this field)
-        <li>amenity - the amenity available at the feature (if applicable), often similar to the category
-        <li>cuisine - the type of food available (if applicable), multiple types are separated by semicolons (;)
-        <li>tourism - the type of tourist location, if applicable (zoo, viewpoint, hotel, attraction, etc.)
-        <li>shop -  the type of shop, if applicable
-        <li>website - the feature's website in the OSM database, if available
-        <li>phone - the feature's phone number(s) in the OSM database, if available
-        <li>open_hours - the feature's operating hours in the OSM database, if available
-        <li>osm_addr - the feature's address in the OSM database, if available
+        <li><code>addr_dist</code> - the distance (m) to the nearest UGRC address point within 25 m

```

Comment:
**thought**: do we want to group the attributes near each other where it makes sense.

It seems addr_dist, ugrc_addr, and osm_addr would logically group together.

Reply:
Yeah, I think that's a better idea